src/formatters/default_formatter.erl

```diff
@@ -309,8 +299,10 @@ lay_no_comments(Node, Ctxt) ->
             Pattern = erl_syntax:match_expr_pattern(Node),
             D1 = lay(Pattern, set_prec(Ctxt, PrecL)),
             D2 = lay(erl_syntax:match_expr_body(Node), set_prec(Ctxt, PrecR)),
-            D3 = case erl_syntax:type(Pattern) == underscore orelse
-                          erl_syntax:type(Pattern) == variable andalso
+            D3 = case erl_syntax:type(Pattern) == underscore
```

Comment:
Assuming this is the formatter's doing, it's odd to have `orelse` and `andalso` isolated like that.

Would it be possible to have

```erlang
            D3 = case erl_syntax:type(Pattern) == underscore
                      orelse erl_syntax:type(Pattern) == variable
                      andalso length(erl_syntax:variable_literal(Pattern)) < Ctxt#ctxt.break_indent
```

?

Reply:
Yeah. That would actually be easier, but I thought that not having an indentation that marks the different _kinds_ of operations would be confusing. What do others think?

Reply:
I think I like the indentation enforcing the precedence rules.